tasks/winbuildscripts/unittests.ps1

```diff
@@ -56,11 +56,8 @@ if($err -ne 0){
     [Environment]::Exit($err)
 }
 
-if ($Env:NEW_BUILDER -eq "true"){
```

Comment:
Is this linked to the change in `.gitlab.yaml` ?

Reply:
It is, this change makes the pipeline use `-race` for every tests on Windows.